OpenRA.Mods.RA/Scripting/Properties/InfiltrateProperties.cs

```diff
@@ -19,18 +22,23 @@ namespace OpenRA.Mods.RA.Scripting
 	[ScriptPropertyGroup("Ability")]
 	public class InfiltrateProperties : ScriptActorProperties, Requires<InfiltratesInfo>
 	{
-		readonly InfiltratesInfo info;
+		readonly IEnumerable<Infiltrates> infiltratesTraits;
 
 		public InfiltrateProperties(ScriptContext context, Actor self)
 			: base(context, self)
 		{
-			info = Self.Info.TraitInfo<InfiltratesInfo>();
+			infiltratesTraits = Self.TraitsImplementing<Infiltrates>();
```

Comment:
Please use `.ToList()` or `.ToArray()`. Otherwise the `IEnumerable` will get evaluated again each time we use it.